src/components/Dialogs/PeriodDimension/PeriodDimension.js

```diff
@@ -145,7 +113,7 @@ export const PeriodDimension = ({ dimension, onClose }) => {
                 if (isStartEndDate(item.id)) {
                     acc.metadata[item.id] = {
                         id: item.id,
-                        name: formatStartEndDate(item.id),
+                        name: item.id.replace('_', ' - '),
```

Comment:
This part is a little bit confusing to me. It seems to me that for these `startEndDates`, we don't actually use the `name` property, but we always use the ID (string) and then parse it. So why do we need to do a string substitution on the `name`?

Reply:
As discussed in a call: this is to keep the metadata consistent. I'll resolve the conversation and approve.